src/bitmessagekivy/tests/test_file_manager.py

```diff
@@ -0,0 +1,117 @@
+"""
+    Test for creating new identity
+"""
+
+from time import sleep
+from random import choice
+from string import ascii_lowercase
+from .telenium_process import TeleniumTestProcess
+from .common import ordered
+
+
+class CreateRandomAddress(TeleniumTestProcess):
+    """This is for testing randrom address creation"""
+    @staticmethod
+    def populate_test_data():
+        pass
```

Comment:
this should populate data instead of repeating the unit tests which already exist in another file.